cmd/kops/update_cluster.go

```diff
@@ -144,6 +144,22 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 		}
 
 		glog.Infof("Using SSH public key: %v\n", c.SSHPublicKey)
+	} else {
+		items, err := keyStore.FindSSHPublicKeys(fi.SecretNameSSHPrimary)
+		if err != nil {
+			return fmt.Errorf("error finding SSH Key: %v", err)
+		}
+		if len(items) == 1 {
+			c.SSHPublicKey = string(items[0].Data)
```

Comment:
Yep you are right - hrmm - it looks like kops doesn't have a concept of where the original key was stored on the users filesystem. Why would it? 

### Option 1

Start tracking the path of the local key in the state store, and use that here. That seems a bit wrong because in theory the path could change, and the state store would have no concept of it.

### Option 2

Change the verbage to say something along the lines of 

``` 
* ssh to the master using the key you provided at creation. Will default to ~/.ssh/id_rsa
* ssh -i $publicKey admin@cluster.com
```

Despite **option 1** feeling a bit wrong, I think I would actually prefer that as a user. I don't change keys that often, and if I did - I would expect some problems in situations like this anyway.

What are your thoughts @justinsb?